pkg/liqoctl/install/provider/provider_suite_test.go

```diff
@@ -19,9 +19,45 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/pflag"
+
+	"github.com/liqotech/liqo/pkg/liqoctl/install/provider"
 )
 
 func TestProvider(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Provider Suite")
 }
+
+func newFlagSet() *pflag.FlagSet {
+	flags := pflag.NewFlagSet("test", pflag.ContinueOnError)
```

Comment:
```suggestion
	flags := pflag.NewFlagSet("test", pflag.PanicOnError)
```

Probably safer.

Reply:
`ContinueOnError` makes `flags.Parse()` return an error (rather than panicking or calling os.Exit), and we don't use flags.Parse in this test anyway.